client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -212,20 +212,15 @@ $font-family: "SF Pro Text", $sans;
 			position: relative;
 			margin-bottom: 32px;
 			padding: 2px;
-			overflow-y: scroll;
```

Comment:
Removing the `overflow-y: scroll` didn't seem to modify the scrolling behavior. I've tested Chrome, Firefox, and Safari with scrollbar always enabled and attached screenshots in the description.